tests/src/androidTest/java/com/percolate/caffeine/testapp/ViewUtilsTest.java

```diff
@@ -125,6 +125,21 @@ public void run() {
         assertEquals(View.GONE, textView.getVisibility());
     }
 
+    public void testHideViewChildView(){
+        final LinearLayout layout = ViewUtils.findViewById(getActivity(), R.id.layout_1);
```

Comment:
Yes. The layout `activity_view_utils ` contains a LinearLayout with id `layout_1`.